code/modules/admin/verbs/freeze.dm

```diff
@@ -6,7 +6,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////
 var/global/list/frozen_mob_list = list()
-/client/proc/freeze(var/mob/living/M as mob in GLOB.mob_list)
+/client/proc/freeze(mob/living/M in GLOB.mob_list)
```

Comment:
Perhaps possible to rewrite this proc to be valid for all atoms?
Seeing the freezemecha proc is on every obj ingame it won't change much I assume.

In the proc here you'd then check the type. mob? Do the usual. Mech? Freeze it's occupant

Reply:
Not a bad idea. Would it be like `atom/movable/M` or something?

Reply:
Yeah something the like. Most detailed path a player can use/be